tests/unit/services/phar/UnixoidPharInstallerTest.php

```diff
@@ -24,7 +24,7 @@
 use PHPUnit_Framework_MockObject_MockObject;
 
 /**
- * @requires OSFAMILY Linux|Darwin
```

Comment:
This syntax is not supported by PHPUnit.
Maybe it would be interesting to write a feature request for it?